### ISSUE TITLE
Code block does not get rendered while starting a marp server with default theme

### ISSUE THREAD

**Issue by djoepramono** [marp-team/marp-cli#189]
Hello, for some weird reason, my code block is no longer working in my browser. It was working a few weeks ago. Would you be able to give me some pointer?

## My markdown presentation

```
---
marp: true
theme: default
---


# Normal

Welcome

---

# Code Block

` ` `js
// apologies for the triple backticks written as so, it's so that it can be previewed properly
console.log('hello');
` ` `
```

## I run the following

```
npx @marp-team/marp-cli -s ./docs
```

## and I got a blank code block like so
![Screenshot from 2019-12-03 00-21-05](https://user-images.githubusercontent.com/2957066/69962795-e28b9c00-1562-11ea-9ff3-36c5cbeb8719.png)

## workarounds I have tried so far
- Change theme to `uncover` and it works, however I can't use that theme because i need a sizeable realestate for my actual code block
- run the cli with `--html` and add 
```
<script defer src="https://cdn.jsdelivr.net/npm/@marp-team/marp-core/lib/browser.js"></script>
``` 

into the code block but it doesn't help

Thank you :pray:

**Comment by djoepramono** [comment]
update: running the marp cli conversion without serving it in a server seems to work

```
@→ npx marp docs/test.md 
[  INFO ] Converting 1 markdown...
[  INFO ] docs/test.md => docs/test.html
```
opening the docs/test.html in a normal browser works fine

**Comment by yhatt** [comment]
It has already reported in #168 and should have fixed in #172 (>= v0.15.0). In fact I could not reproduce the reported bug.

What the version of Marp CLI are you using? (`--version`)

**Comment by djoepramono** [comment]
Ah apologies my `marp` is still locked in version 14.x.

Thanks, I'm closing this.
